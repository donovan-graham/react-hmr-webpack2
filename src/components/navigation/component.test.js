import React from 'react';
import Nav from './component';
import renderer from 'react-test-renderer';

const noOp = () => null;

describe('Navigation component', () => {
  it('main naviation', () => {
    const tree = renderer.create(<Nav />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('navigation link', () => {
    const tree = renderer.create(<Nav.Link href="/">Home</Nav.Link>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
