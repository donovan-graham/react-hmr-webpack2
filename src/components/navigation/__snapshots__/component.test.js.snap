// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation component main naviation 1`] = `
<nav
  className="fuzYCF"
/>
`;

exports[`Navigation component navigation link 1`] = `
<a
  className="cbSlSU"
  href="/"
  onClick={[Function]}
  style={Object {}}
>
  Home
</a>
`;
